lib/charms/layer/hacluster.py

```diff
@@ -0,0 +1,41 @@
+from charms.reactive import clear_flag, set_flag
+from charmhelpers.core import unitdata
+
+db = unitdata.kv()
+
+
+def add_service_to_hacluster(name, service_name):
+    """Adds a service to be monitored under HAcluster.
+       Takes a name for this entry and a service_name,
+       which is the name used by systemd/initd to identify
+       the service.
+    """
+    services = db.get('services', {'current_services': {},
```

Comment:
It may be worthwhile to rename this database key to `layer-hacluster.services` to avoid conflicts with other layers.

Reply:
Yes, I agree.